# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: D:\Code\SVN\trunk\menu_token
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: menu_token.module
--- menu_token.module 基本 (BASE)
+++ menu_token.module 本地修改 (基于LOCAL)
@@ -115,7 +115,7 @@
 
     // If item is generated by admin menu module, tokens should not be replaced
     // and indicator that tokens are used should be shown.
-    $item['title'] = token_replace($item['title'], $data, $options);
+    $item['title'] = token_replace($item['link_title'], $data, $options);
     $url = token_replace($item['options']['menu_token_link_path'], $data, $options);
 
     // Make sure aliases are proccessed correctly
